File: CS101计算机编程基础系列/02_变量标识符表达式/Cpp/CalculateArea.cpp
#include<iostream>
#include<cmath>

using namespace std;

int main(){
    printf("%s", "请输入两个整数，用空格隔开：");
    int num1, num2;
    scanf("%d %d", &num1, &num2);

    /*cout << "请输入第二个整数：";
    int num2;
    cin >> num2;*/

    //cout << "两个整数的和是：" << num1 + num2 << endl;

    printf("两个整数的和是：%d\n", (num1 + num2));

    cout << "请输入圆的半径: ";
    double radius = 0.0;
    cin >> radius;

    double area = M_PI * pow(radius, 2);
    cout << "圆的面积是：" << area << endl;

    system("pause");
    return 0;
}

File: CS101计算机编程基础系列/02_变量标识符表达式/Java/ExpressionDemo/src/CalculateArea.java
/*
 * 本程序接收两个整数和一个浮点数作为输入
 * 输出这两个整数之和
 * 和这个浮点数为半径的圆的面积。
 * 
 */

import java.util.Scanner;

public class CalculateArea {
    public static void main(String[] args) {
        
        System.out.print("Input integer 1: ");
        
        Scanner input = new Scanner(System.in);
        int num1 = input.nextInt();
        
        System.out.print("Input integer 2: ");
        int num2 = input.nextInt();

        System.out.println("num1 + num2 = " + (num1 + num2));

        System.out.print("Input the radius: ");
        double radius = input.nextDouble();
        double area = Math.PI * Math.pow(radius, 2);
        System.out.println("The area of the circle is: " + area);

    }
    
}
